wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -376,8 +388,14 @@ instance ToSchema (V1 Core.Timestamp) where
 -- base16-encoded string.
 type SpendingPassword = V1 Core.PassPhrase
 
+instance Monoid (V1 Core.PassPhrase) where
+    mempty = V1 mempty
+    mappend (V1 a) (V1 b) = V1 (a `mappend` b)
```

Comment:
Why do we need to `mappend` two `PassPhrase`(s) together?

Reply:
I haven't seen a case where we need the SemiGroup part of it, but we *require* `mempty` in a couple of places. I use it as a *default value* to get rid of the `Maybe PassPhrase` we have sometimes when securely logging those phasephrase. In the end, the value doesn't matter much and could be anything since we display a placeholder `<passphrase>` anyway. 

Though, I noticed a function `emptyPassphrase` in one of the core type and I wondered why not using well-known abstractions instead (like Monoid).

Reply:
Fair enough. I still have in the back of my mind that concatenating two `PassPhrase`(s) makes no sense at all and thus it shouldn't be allowed, but I don't think that's the end of the world, as the type has a well understood semantic.